File: ffrab.droid/MainActivity.fs
namespace ffrab.mobile.droid

open System

open Android.App
open Android.Content
open Android.OS
open Android.Runtime
open Android.Views
open Android.Widget
open Xamarin.Forms
open Xamarin.Forms.Platform.Android

[<Activity (Label = "ffrab.droid", MainLauncher = true)>]
type MainActivity () =
    inherit FormsApplicationActivity ()


    override this.OnCreate (bundle) =

        base.OnCreate (bundle)

        Forms.Init (this, bundle)

        let sqlPlatform = new SQLite.Net.Platform.XamarinAndroid.SQLitePlatformAndroid()
        let databasePath = System.IO.Path.Combine(System.Environment.GetFolderPath(Environment.SpecialFolder.Personal), "ffrab.mobile.db")

        
        let app = new ffrab.mobile.common.app.App(sqlPlatform, databasePath)

        this.LoadApplication(app)

File: ffrab.droid/MainActivity.cs
﻿using System;
using Android.App;
using Android.OS;
using Xamarin.Forms;
using Xamarin.Forms.Platform.Android;

namespace www.linuxwochen.droid
{
	[Activity(Label = "Linuxwochen Wien", Icon = "@mipmap/icon", 
	          ConfigurationChanges=Android.Content.PM.ConfigChanges.Orientation, ScreenOrientation= Android.Content.PM.ScreenOrientation.Portrait)]
	public class MainActivity : FormsApplicationActivity
	{
		protected override void OnCreate(Bundle savedInstanceState)
		{
			base.OnCreate(savedInstanceState);

			Forms.Init(this, savedInstanceState);

			var databasePath = System.IO.Path.Combine(System.Environment.GetFolderPath(System.Environment.SpecialFolder.Personal), "linuxwochen.db");

			var app = new www.linuxwochen.common.app.App(databasePath);

			this.LoadApplication(app);
		}
	}
}


